it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala

```diff
@@ -151,8 +154,8 @@ class ExciseNumberQueueWorkItemRepositoryItSpec extends PlaySpec
   private def createWorkItemForErn(ern: String, updateAt: Instant = timestamp)  =
```

Comment:
updateAt argument is not used, can we remove it please? I know this was already there,

Reply:
Done